### ISSUE TITLE
[findrepe] findrepe do not exclude paths in windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#8]
```
findrepe do not exclude paths in windows.
```

Original issue reported on code.google.com by `flik...@gmail.com` on 2 Jul 2009 at 5:19

**Comment by GoogleCodeExporter** [comment]
```
fixed, version 0.6.0
```

Original comment by `flik...@gmail.com` on 4 Jul 2009 at 12:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 10 Jan 2010 at 11:17
- Changed title: **[findrepe] findrepe do not exclude paths in windows**
